packages/core/src/components/DatePicker/DatePicker.tsx

```diff
@@ -115,46 +125,69 @@ export const DatePicker: React.FC<Props> & WithStyle = React.memo(
 
         const suffixEl = () => (
             <DateIcon variant={restProps.variant} isErrorPresent={isErrorPresent} isActive={active} disabled={disabled} size={size}>
-                <DateRangeIcon id={`${id}-calendar-icon`} onClick={onIconClick} size={size} />
+                <DateRangeIcon onClick={onIconClick} size={size} />
```

Comment:
Why you removed id ?

Reply:
This is a mistake, thanks for pointing it out!

Reply:
https://github.com/medly/medly-components/pull/186/commits/b5ce89048edb4be4e304190ee07ccdc707847bc2